Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,46 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main()
         {
-            Console.WriteLine("Hello World!");
+            // Initialising the type of logger for the entire application
+            ILogger typeoflogger = InitialiseLogger();
+
+            // Simulating the logger through using a Calculator Sim
+            var calculate = new CalculatorSimulator(typeoflogger);
+
+            // hardcoded values while de-bugging
+            calculate.Add(1, 10);
+            calculate.Subtract(10, 9);
+            Console.Read();
+        }
+
+        // enum for Logging Options to make code more readable
+        enum LoggingOptions
+        {
+            File = 1,
+            Console = 2
+        }
+
+        private static ILogger InitialiseLogger()
+        {
+            while (true)
+            {
+                Console.WriteLine("Would you like to create a Log File (1) or Log in Console (2) to track activity?");
```

Comment:
The two lines are a bit out of place. what would you do, if you needed to create a logger without writing to a console. You should split this method into two:
```
private static ILogger InitializeLogger()
{
   LoggingOptions options = PromptLoggingOptions();
   ILogger logger = CreateLogger(options);
   
    return logger; 
}

private static ILogger CreateLogger(){...}
private static LoggingOptions PromptLoggingOptions(){...}
```